### PR TITLE
added scottish lidar yaml file

### PR DESCRIPTION
**Pull request by foxfoxfox7**
*Issue #, if available:*

*Description of changes:*
The yaml file for the Scottish lidar dataset.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/scottish-lidar.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  Tutorials:
+    - Title: LiDAR Tutorial using QGIS
+      URL: https://datasciencescotland.github.io/LiDAR-Tutorial-QGIS/
+      AuthorName: Michal Michalski
+    - Title: LiDAR Tutorial using R
+      URL: https://datasciencescotland.github.io/LiDAR-Tutorial-R/
+      AuthorName: Michal Michalski
+  Tools & Applications:
+    - Title: Scottish Remote Sensing Portal
+      URL: https://remotesensingdata.gov.scot/
```
Comment (jeffejefe) [inline datasets/scottish-lidar.yaml:30]: @foxfoxfox7 can you add AuthorName/URL and then resubmit your PR (verify if correct also). It is required for dataset onboarding.
```
AuthorName: Scottish Government and Joint Nature Conservation Committee (JNCC)
AuthorURL: https://remotesensingdata.gov.scot/about
```

**Review by jeffejefe — COMMENTED** [review]
Thanks @foxfoxfox7 just had a comment under Tools & Applications.

**Comment by jeffejefe** [comment]
Thanks for quickly updating @foxfoxfox7. Looking forward to working more with this dataset!

**Comment by foxfoxfox7** [comment]
Thanks for all your help! :)

**Pull request merged**